### PR TITLE
Small fixes...

### PR DESCRIPTION
**Pull request by white238**
* Silence a policy
* Simplify CUDA+OpenMP smoke test (#296 #297) and moves the current smoke test to a properly guarded internal test

### CODE REVIEW

File: thirdparty_builtin/googletest-master-2018-02-01/googletest/CMakeLists.txt [hunk 1]
```diff
@@ -61,9 +61,11 @@ if (POLICY CMP0063) # Visibility
   cmake_policy(SET CMP0063 NEW)
 endif (POLICY CMP0063)
 
-if (POLICY CMP0042) # MACOSX_RPATH behavior
-  cmake_policy(SET CMP0042 OLD)
-endif (POLICY CMP0042)
+if (APPLE)
```
Comment (rhornung67) [inline thirdparty_builtin/googletest-master-2018-02-01/googletest/CMakeLists.txt]: 👍

File: tests/internal/CMakeLists.txt [hunk 2]
```diff
@@ -33,7 +33,7 @@ option(TEST_GIT_MACROS
 # Add a library
 #------------------------------------------------------------------------------
 
-blt_add_library( NAME example
+blt_add_library( NAME    example
```
Comment (davidbeckingsale) [inline tests/internal/CMakeLists.txt]: Exquisite spacing 👍

Comment (white238) [reply tests/internal/CMakeLists.txt]: i am amazing...

Comment (rhornung67) [reply tests/internal/CMakeLists.txt]: 🥇

**Comment by rhornung67** [comment]
@white238 when do you plan to merge this?

**Pull request merged**